### ISSUE TITLE
Erro make 

### ISSUE THREAD

**Issue by aderbas** [ardje/vizzini#1]
I'm try to make driver but gets this error

```
$ make[1]: *** /lib/modules/4.4.38-v7+/build: No such file or directory.  Stop.
$ Makefile:12: recipe for target 'modules_install' failed
$ make: *** [modules_install] Error 2
```

Using Raspberry Pi with Raspbean
`Linux raspberrypi 4.4.38-v7+ #938 SMP Thu Dec 15 15:22:21 GMT 2016 armv7l GNU/Linux`

**Comment by bachphi** [comment]
same here:
make -C /lib/modules/4.4.38-v7+/build M=/home/pi/tty0tty-1.2/module modules
make[1]: *** /lib/modules/4.4.38-v7+/build: No such file or directory.  Stop.
Makefile:26: recipe for target 'default' failed
make: *** [default] Error 2

where is the build?

**Comment by ardje** [comment]
Oh, wow... I got too many notifications...
Sorry.
I've archived vizzini on github, as there is another version:
https://github.com/ardje/exar-uart-driver
Of which the upstream:
https://github.com/shenki/exar-uart-driver
is more up-to-date.

Furthermore: I don't think you are compiling things using make-kpkg...
Anyway: vizzini is incompatible with newer kernels, the exar-uart driver also works on 4.9. So I close this ticket.
